Core/src/org/sleuthkit/autopsy/report/ReportHTML.java

```diff
@@ -430,6 +452,40 @@ public void endDataType() {
             }
         }
     }
+    
+    /**
+     * Write HTML-formatted page header text based on the text provided in the
+     * configuration panel.
+     * 
+     * @return The HTML-formatted text.
+     */
+    private String writePageHeader() {
+        StringBuilder output = new StringBuilder();
+        String pageHeader = configPanel.getHeader();
+        if (pageHeader.isEmpty() == false) {
+            output.append("<p style=\"text-align: center\">")
```

Comment:
So I would kinda like to see the size of this text a little bigger it sounded like Dimitry thought it should be about the same size as the section heading, and if we are going to do that I think we should define the size and style stuff in the CSS we write around lines 1130 for the summary and lines 915 for the sections.

Something like 
"#headerFooter {width:100%; padding: 10px; line-height: 25px; text-align: center; font-size: 20px;}\n"

So that in the write page header and write page footer code we would be appending "<div id=\"headerFooter\">"  before the text and "</div>\n" after the text.